modules/__init__.py

```diff
@@ -1,38 +1,28 @@
 from modules.base import Module, noparts, check_solve_cmd, gif_append, gif_output
+import glob
+from os.path import join as pjoin
+from os.path import dirname as pdir
+from os.path import basename as pbase
+import importlib
 
-from modules.wires import Wires, ComplicatedWires, WireSequence
-from modules.button import Button
-from modules.keypad import Keypad
-from modules.simon import SimonSays
-from modules.whos_on_first import WhosOnFirst
-from modules.memory import Memory
-from modules.morse import MorseCode
-from modules.maze import Maze
-from modules.password import Password
+VANILLA_MODULE_LIST="wires/button/keypad/simonSays/whosOnFirst/memory/morseCode/complicatedWires/wireSequence/maze/password".split("/")
```

Comment:
I think it would've been better to add a `vanilla` attribute to modules (like `display_name` or `module_score` are now). The loader could then check it and add the module to the right category. Adding `vanilla = False` in the base class will give it a default value.

Reply:
That would work better :D